doc/nrf/ug_nrf91_features.rst

```diff
@@ -160,6 +160,26 @@ The :ref:`nrfxlib:nrf_modem` is released as an OS-independent binary library in
 The Modem library integration layer fulfills the integration requirements of the Modem library in |NCS|.
 For more information on the integration, see :ref:`nrf_modem_lib_readme`.
 
+.. _modem_trace:
+
+Modem trace
+-----------
+
+The modem traces of the nRF9160 modem can be captured using the nRF Connect Trace Collector.
+For more information on how to collect traces using nRF Connect Trace Collector, see the `Trace Collector`_ documentation.
+When the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE` Kconfig option is enabled, the modem traces are enabled in the modem and are forwarded to the :ref:`modem_trace_module`.
+
+.. note::
+   For the :ref:`serial_lte_modem` application and the :ref:`at_client_sample` sample, you must also run ``AT%xmodemtrace=1,2`` to manually activate the predefined trace set.
+
+You can set the trace level using the AT command ``%XMODEMTRACE``.
+See `modem trace AT command documentation`_ for more information.
+
+You can add custom trace backends if the existing trace backends are not sufficient.
+The modem traces can be stored, flashed, and uploaded to the cloud for analysis whenever required.
+For more information on the implementation of a custom trace backends, see :ref:`adding custom trace backends <adding_custom_modem_trace_backends>`.
+You can also add a user-defined modem trace backend to an application using the :ref:`modem_trace_backend_sample` sample.
```

Comment:
The «Adding custom modem trace backends» mentions and links to the sample, so I don't think it's necessary to mention it here in addition to what's already said.
```suggestion
```